### PR TITLE
{bio,vis}[foss/2021a] Scoary v1.6.16, ETE v3.1.2 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/Scoary/Scoary-1.6.16-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+use_pip = True
+
+exts_list = [
+    ('argparse', '1.4.0', {
+        'checksums': ['62b089a55be1d8949cd2bc7e0df0bddb9e028faefc8c32038cc84862aefdd6e4'],
+    }),
+    (name, version, {
+        'source_tmpl': 'scoary-%(version)s.tar.gz',
+        'checksums': ['f433343422c5805a70a20f7fe83851490ae3cdf07745c65680de5feea40a6dca'],
+        'modulename': False,
```
Comment (smoors) [inline easybuild/easyconfigs/s/Scoary/Scoary-1.6.16-foss-2021a.eb:30]: @verdurin why set it to False? maybe set it to `scoary.methods` ?

Comment (verdurin) [reply easybuild/easyconfigs/s/Scoary/Scoary-1.6.16-foss-2021a.eb]: @smoors fixed, and `sanity_check_command` added.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
node360.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/b15695be1cebdd1b085c1732358b2158 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @verdurin!

**Pull request merged**